lib/terminus.js

```diff
@@ -3,26 +3,29 @@
 const stoppable = require('stoppable')
 const { promisify } = require('es6-promisify')
 
-const SUCCESS_RESPONSE = JSON.stringify({
+const SUCCESS_RESPONSE = {
   status: 'ok'
-})
+}
 
-const FAILURE_RESPONSE = JSON.stringify({
+const FAILURE_RESPONSE = {
   status: 'error'
-})
+}
 
 function noopResolves () {
   return Promise.resolve()
 }
 
-function sendSuccess (res) {
+function sendSuccess (res, info) {
   res.statusCode = 200
-  res.end(SUCCESS_RESPONSE)
+  if (info) {
+    SUCCESS_RESPONSE.info = info
+  }
+  res.end(JSON.stringify(SUCCESS_RESPONSE))
```

Comment:
if there is no `info` provided, can we used the cached JSON.stringify result, so it won't get stringified every time again?